.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Report
+about: You have noticed a general issue or regression, and would like to report it
+labels: bug
+---
+
+**Describe The Bug**
+
+<!-- A clear and concise description of what the bug is. -->
```

Comment:
```suggestion
<!-- A clear and concise description of the bug. -->
```
This sounds better to me, although I'm not an English speaker so my opinion doesn't matter too much. What do you think?

Reply:
Yeah that's a better description 👍